server/lib/service/firestore.dart

```diff
@@ -63,6 +67,29 @@ class FirestoreClient {
     );
   }
 
+  /// Logs [isSuccess] and optional [message] of a performed rule with [ruleId].
+  ///
+  /// The [userId] is the `sub` claim of the user's Google ID token.
+  /// See: https://developers.google.com/identity/protocols/oauth2/openid-connect#an-id-tokens-payload
+  ///
+  /// Throws an [ApiRequestError] if Firestore API returns an error.
+  Future<void> addRun(String userId, String ruleId,
+      {@required bool isSuccess, String message = ''}) async {
```

Comment:
I think moving `bool isSuccess` out of the `{}` would indicate this param is required.

Reply:
Ok, that makes sense. I did it this way because one of Aqueduct's default lint rules is to [avoid positional boolean parameters](https://dart-lang.github.io/linter/lints/avoid_positional_boolean_parameters.html).

Reply:
If that is the case, you can keep what you had before.